requirements/app.in

```diff
@@ -17,7 +17,7 @@ psycopg2-binary
 bcrypt
 pytz
 # limit the numpy version to make it compatible with numba==0.56.4, which timely-beliefs >=1.18 depends on (library sktime).
```

Comment:
Maybe this comment needs an update, regarding the numba compatibility

Reply:
Specifically, what numba version do we need this for (it probably changed)?
If we don't know and don't have time to find out, let's simplify this comment to what we know: timeley_beliefs -> sktime

Reply:
From the official repo of `sktime` I believe numba is an optional dependency for `sktime`

https://github.com/sktime/sktime/blob/9d6c856825e4272326931b3aa01ecb744fda9478/pyproject.toml#L56.